install/kubernetes/quick-install.yaml

```diff
@@ -1,18 +1,4 @@
 ---
-# Source: cilium/charts/agent/templates/serviceaccount.yaml
```

Comment:
We might have a problem, for some reason, on my system theses files are moved to this place again. Which helm version are you using to generate this file?

Reply:
`version.BuildInfo{Version:"v3.0.3", GitCommit:"ac925eb7279f4a6955df663a0128044a8a6b7593", GitTreeState:"clean", GoVersion:"go1.13.6"}`

Reply:
Regarding Helm version requirements: #10418

Reply:
I have:
```
$ helm version
version.BuildInfo{Version:"v3.1.1", GitCommit:"afe70585407b420d0097d07b21c47dc511525ac8", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

Reply:
Ok, I'll update helm and re-generate again.

Reply:
Pushed a new version generated with the same helm version you have. Now the files are not moved anymore.